config/secrets.defaults.yml

```diff
@@ -2,6 +2,14 @@
 test:
   ibm_cloud_power_defaults: &ibm_cloud_power_defaults
     api_key: IBM_CLOUD_POWERVS_API_KEY
+    power_instance_id: IBM_CLOUD_POWERVS_INSTANCE_ID 
```

Comment:
I think Travis failed because this default was added after recording the refresher spec VCR cassette.

>        An HTTP request has been made that VCR does not know how to handle:
>         GET https://resource-controller.cloud.ibm.com/v2/resource_instances/IBM_CLOUD_POWERVS_INSTANCE_ID

 There's [code in spec_helper.rb](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/spec/spec_helper.rb#L77-L79) that will replace the VCR cassette values with your default value. Since you recorded and then afterwards added these default, I think the replacement never occurred.

Reply:
I re-recorded the VCRs and the powervs instance was replaced as expected, committed and pushed

Reply:
Adam, Thanks. Jay thanks for clarifying. I have rerecorded and pushed the code.